Makefile

```diff
@@ -1,193 +0,0 @@
-.PHONY: all
```

Comment:
Why does it show that his Makefile was deleted and recreated in the tracee-ebpf dir?
Can you use git mv instead?